src/service/emailService.js

```diff
@@ -427,7 +439,221 @@ function publishedContentEmail (req, callback) {
+  if (lodash.includes(data.roles, 'CONTENT_CREATOR')) {
+    fetchDetailsFlag = false
+  }
+  var subOrgReviewer = {
+    'request': {
+      'filters': {
+        'organisation.organisationId': lodash.uniq(orgIds),
+        'organisations.roles': ['CONTENT_REVIEWER']
+      },
+      'limit': 200,
+      'offset': 0
+    }
+  }
+  async.parallel({
+    rootOrgReviewers: getUserIds(req, rootOrgReviewer, true),
+    subOrgReviewers: getUserIds(req, subOrgReviewer, fetchDetailsFlag)
+  }, function (err, results) {
+    if (err) {
+      callback(err, null)
+    } else {
+      var rootOrgReviewersId = lodash.map(results.rootOrgReviewers, 'id')
+      var subOrgReviewersId = lodash.map(results.subOrgReviewers, 'id')
+      var allReviewerIds = lodash.union(rootOrgReviewersId, subOrgReviewersId)
+      callback(null, allReviewerIds)
+    }
+  })
+}
+
+/**
+ * Below function is used to get reviewer ids recursively if count is more than 200
+ * @param {object} req
+ * @param {object} body
+ * @param {function} callback
+ */
+function getUserIds (req, body, fetchDetailsFlag) {
+  if (fetchDetailsFlag) {
+    return function (CBW) {
+      var totalCount = 0
+      async.waterfall([
+        function getFirst200 (callback) {
+          contentProvider.userSearch(body, req.headers, function (err, result) {
+            if (err || result.responseCode !== responseCode.SUCCESS) {
+              callback(new Error('User Search failed'), null)
+            } else {
+              callback(null, {count: result.result.response.count, content: result.result.response.content})
+            }
+          })
+        },
+        function recursiveUserCalling (data, callback) {
+          if (data.count < 200) {
+            callback(null, data.content)
+          } else {
+            var userDetails = data.content
+            totalCount = data.count / 200
+            var parallelFunctions = []
+            for (var i = 1; i <= totalCount; i++) {
+              var parallelFun = function (request) {
+                return function (callback1) {
+                  contentProvider.userSearch(request, req.headers, function (err, result) {
+                    if (err || result.responseCode !== responseCode.SUCCESS) {
+                      callback1(new Error('User Search failed'), null)
+                    } else {
+                      callback1(null, result.result.response.content)
+                    }
+                  })
+                }
+              }
+              var reqBody = lodash.cloneDeep(body)
+              reqBody.request.offset = 25 * i
+              parallelFunctions.push(parallelFun(reqBody))
+            }
+            async.parallel(parallelFunctions, function (err, data) {
+              if (err) {
+                callback(new Error('User Search failed'), null)
+              } else {
+                var userData = []
+                lodash.forEach(data, function (value) {
+                  lodash.forEach(value, function (res) {
```

Comment:
lodash.forEach(value, function (res) {
                    userData.push(res)
                  })
to 
userDetails.concat(value)